examples/platform/nrfconnect/util/include/NFCWidget.h

```diff
@@ -27,11 +27,14 @@ class NFCWidget
     int Init(chip::DeviceLayer::ConnectivityManager & mgr);
     int StartTagEmulation(const char * tagPayload, uint8_t tagPayloadLength);
     int StopTagEmulation();
+    bool GetTagState();
```

Comment:
`GetTagState()` suggests some state will be returned, while what is returned is boolean value indicating if TAG emulation is turned on/off. What about to call it `IsTagStarted()` or `IsTagEmulationStarted()`, alternatively start using Enable/Disable names in API?

Reply:
Also change to bool `GetTagState() const`. Maybe even worth concerning implementing it in `.h` file directly.